openfasoc/generators/glayout/glayout/flow/blocks/composite/regulated_cascoded_current_mirror/regulated_cascoded_current_mirror.py

```diff
@@ -0,0 +1,374 @@
+import sys
+from os import path, rename, environ , listdir, remove, chmod
+# environ['OPENBLAS_NUM_THREADS'] = '1'
+from pathlib import Path
+# path to glayout
+sys.path.append(path.join(str(Path(__file__).resolve().parents[2])))
+
+from glayout.flow.pdk.mappedpdk import MappedPDK
+from glayout.flow.pdk.sky130_mapped import sky130_mapped_pdk as sky130
+from glayout.flow.pdk.gf180_mapped import gf180_mapped_pdk as gf180
+
+from glayout.flow.primitives.guardring import tapring
+from glayout.flow.routing.smart_route import smart_route
+from glayout.flow.routing.straight_route import straight_route
+from glayout.flow.routing.c_route import c_route
+from glayout.flow.routing.L_route import L_route
+
+from glayout.flow.placement.two_transistor_interdigitized import two_nfet_interdigitized, two_pfet_interdigitized
+
+from glayout.flow.pdk.util.snap_to_grid import component_snap_to_grid
+from glayout.flow.pdk.util.comp_utils import evaluate_bbox, prec_center, prec_array, movey, align_comp_to_port, prec_ref_center
+from glayout.flow.pdk.util.port_utils import rename_ports_by_orientation, rename_ports_by_list, add_ports_perimeter, print_ports, set_port_orientation, rename_component_ports
+
+from gdsfactory.components import text_freetype, rectangle
+from gdsfactory import Component
+from gdsfactory.routing.route_quad import route_quad
+from glayout.flow.spice.netlist import Netlist
+from typing import Optional, Union 
+
+def delete_files_in_directory(directory_path):
```

Comment:
This is not related to the current mirror and should be in a different file if used for testing.